srcpkgs/boot-66serv/files/66

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/execlineb -P
+66-boot -b "Booting Voidlinux" -m
```

Comment:
VoidLinux or "Void Linux" would be preferred, I think

Reply:
Void Linux is in the site, so I will change to that.

Reply:
Done, thank you!